lib/modules/versioning/cargo/index.ts

```diff
@@ -1,9 +1,16 @@
+import semver from 'semver';
+import stable from 'semver-stable';
+
 import { logger } from '../../../logger';
 import type { RangeStrategy } from '../../../types/versioning';
 import { regEx } from '../../../util/regex';
 import { api as npm } from '../npm';
 import type { NewValueConfig, VersioningApi } from '../types';
 
+const { is: isStable } = stable;
+
+const { major: getMajor, minor: getMinor } = semver;
```

Comment:
why not doing this as part of the import?

Reply:
Done